src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -2481,7 +2481,8 @@ public void Optimize()
         {
             var parsedArgs = DefaultParse(new[] { "a.cs" }, WorkingDirectory);
             parsedArgs.Errors.Verify();
-            Assert.Equal(new CSharpCompilationOptions(OutputKind.ConsoleApplication).OptimizationLevel, parsedArgs.CompilationOptions.OptimizationLevel);
+            // TODO: I think the test is more clear with the old approach (explicitly creating CSharpCompilationOptions). What do you think?
```

Comment:
The warning level doesn't matter in this specific test I think. Should I keep using the old approach? or keep the new approach?

Reply:
IMO use the old approach